style.css

```diff
@@ -0,0 +1,88 @@
+body{
+    font-family: 'Poppins', sans-serif;
+    font-size: 1 rem;
+    font-weight: 400;
+    color: #fff;
+    background-image: 
+    linear-gradient(
+        115deg,
+        rgba(58, 58, 158, 0.8),
+        rgba(136, 136, 206, 0.7)
+      ),
+      url(https://cdn.freecodecamp.org/testable-projects-fcc/images/survey-form-background.jpeg)
+    
+      
+    }
+
```

Comment:
- [ ] As in html Remove unnecessary spaces to make the code more readable.
- [ ] Remember to use indentation to make your code clearer.